src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,60 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+
 public class WorkWithFile {
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String RESULT = "result";
+    private static final int FIELD_INDEX = 0;
+    private static final int VALUE_INDEX = 1;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        File toFile = new File(toFileName);
+        if (isFileEmpty(toFile)) {
+            try (FileWriter fileWriter = new FileWriter(toFile)) {
+                fileWriter.write(prepareReport(readFile(fromFileName)));
+            } catch (IOException e) {
+                throw new RuntimeException("File doesn't exist " + toFileName);
+            }
+        } else {
+            System.out.println("File isn't empty. Delete data.");
+        }
```

Comment:
make `getStatistic `like this:
```suggestion
         String dataFromFile = readFile(fromFileName);
         String report = createReport(dataFromFile);
         writeToFile(report, toFileName);
```